rpm/nginx-ldap-auth.spec

```diff
@@ -50,6 +51,7 @@ getent passwd nginx-ldap-auth > /dev/null || \
 %preun
 /usr/bin/systemctl --no-reload disable nginx-ldap-auth.service >/dev/null 2>&1 ||:
 /usr/bin/systemctl stop nginx-ldap-auth.service >/dev/null 2>&1 ||:
+rm -f /var/log/nginx-ldap-auth/*
```

Comment:
There is no check if this is an upgrade or removal. Either way it looks to be a bad idea to purge all logs.

Reply:
I wasn't too sure about this (or spec files in general), so went looking for examples.  I got this from https://github.com/mozilla-services/squid-rpm/blob/ea9a11c15092788f270730e0264f01ab5503a82e/SPECS/squid.spec#L253
But maybe that's not a very good practice and I'm cargo-cult copying a bad example!

Reply:
Just come across https://fedoraproject.org/wiki/PackagingDrafts/Logfiles

> logfiles should not be removed by package removals

I can update the PR to follow those guidelines.